transformer_engine/pytorch/fp8.py

```diff
@@ -64,13 +64,22 @@ def check_fp8_block_scaling_support() -> Tuple[bool, str]:
     return False, "FP8 block scaled GEMM requires Hopper and CUDA >= 12.9."
 
 
+def check_recipe_support(recipe: Recipe) -> None:
+    """Check if the given recipe is supported."""
+    recipe_supported = False
```

Comment:
https://github.com/NVIDIA/TransformerEngine/pull/2039 is adding support for custom user-provided recipes. We should presume that recipes are supported and put the burden on users if their implementation has hardware limitations:
```suggestion
    recipe_supported = True
```